packages/bpk-component-label/src/BpkLabel.js

```diff
@@ -26,6 +26,7 @@ import STYLES from './BpkLabel.scss';
 const getClassName = cssModules(STYLES);
 
 export type Props = {
+  // $FlowIssue[value-as-type] - Node is a type import from prop-types and is mishandled by Flow
```

Comment:
An example where a new issue has been introduced from the new version.

We could potentially solve this by changing the import to

```js
import { Node } from 'prop-types'
```

and then do 

```js
children: typeof Node
```

but not sure if that is what we should do vs this being an issue with Flow?

Reply:
Just wanna check, as this might be a legit bug we're covering up — usually we do `import { type Node } from 'react'`, i.e. not `'prop-types'`. I've never seen it imported from prop-types so I think it's a mistake, although I might be wrong.

If you change it to import from `react` does this error go away?

Reply:
Will have a look!

Reply:
This might be from an old implementation before React had support so if that solves it and we can remove this line that will be perfect!

Reply:
🤞

Reply:
Yep looks like this was fine to change to 😄